polynote-frontend/polynote/cell.js

```diff
@@ -195,6 +195,10 @@ export class Cell extends UIEventTarget {
         ])
     }
 
+    setDisabled(disabled) {
+
```

Comment:
do you want to throw a not implemented error (like `Cell.applyEdits` does) for this?

Reply:
Not really